### PR TITLE
acceptance: Add accepted error to TestUpdate

### LINKED ISSUE #6715: circleci: failed tests: TestRaftUpdate
The following test appears to have failed:

[#17861](https://circleci.com/gh/cockroachdb/cockroach/17861):

```
I160516 19:31:04.465875 acceptance/replication_test.go:66  found 3 replicas
I160516 19:31:04.465903 acceptance/replication_test.go:74  found 3 replicas
I160516 19:32:04.914642 acceptance/cluster/localcluster.go:658  stopping
I160516 19:32:04.914879 acceptance/cluster/localcluster.go:556  event stream done, resetting...: net/http: request canceled
I160516 19:32:04.914899 acceptance/cluster/localcluster.go:579  events monitor exits
--- FAIL: TestRaftUpdate (65.69s)
	update_test.go:54: acceptance/update_test.go:63: Post https://127.0.0.1:32885/_admin/v1/cluster/freeze: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
FAIL
ok github.com/cockroachdb/cockroach/acceptance 1337s

```
Please assign, take a look and update the issue accordingly.

### PR DESCRIPTION
**Pull request by tbg**
Increasing the timeout caused another error to pop up.
Last attempt to de-flake that test before disabling it for now.

Closes #6715.

---

This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/6724)

### CODE REVIEW

File: acceptance/update_test.go [hunk 1]
```diff
@@ -127,7 +133,7 @@ func testRaftUpdateInner(t *testing.T, c cluster.Cluster, cfg cluster.TestConfig
 	}
 
 	// Unfreezing again should be a no-op.
-	if reply, err := postFreeze(c, false); err != nil {
+	if reply, err := postFreeze(c, false, long); err != nil {
```
Comment (tamird) [inline acceptance/update_test.go]: i think you mixed long and short - wouldn't this no-op be the fast one?

File: acceptance/update_test.go [hunk 2]
```diff
@@ -86,7 +90,9 @@ func testRaftUpdateInner(t *testing.T, c cluster.Cluster, cfg cluster.TestConfig
 	// of the last node receiving the freeze command).
 	// Note that this is the freeze trigger stalling on the Replica, not the
 	// Store-polling mechanism.
-	if reply, err := postFreeze(c, true); !testutils.IsError(err, "timed out waiting for Range|Timeout exceeded while") {
+	acceptErrs := strings.Join([]string{"timed out waiting for Range",
```
Comment (tamird) [inline acceptance/update_test.go]: nit this is hard to read. how about
```
strings.Join([]string{
"foo",
"bar",
}, "|")
```

File: acceptance/update_test.go (hunk 1 above)
Comment (tschottdorf) [inline acceptance/update_test.go]: I want only the command which I'm expecting to time out to actually time out (but I'm not ready to wait a minute). For everything else, I want to reduce spurious errors, so a long timeout.

File: acceptance/update_test.go (hunk 2 above)
Comment (tschottdorf) [inline acceptance/update_test.go]: I see your point, but given that this test fails pretty reliably on master, I don't want to invest another CI period.

Comment (tamird) [inline acceptance/update_test.go]: sgtm. 🚢

**Comment by bdarnell** [comment]
LGTM

<details><summary><i>Previously, tschottdorf (Tobias Schottdorf) wrote…</i></summary>
> #### acceptance: Add accepted error to TestUpdate
> 
> Increasing the timeout caused another error to pop up.
> 
> Last attempt to de-flake that test before disabling it for now.
> 
> 
> 
> Closes #6715.
> 
> 
> </details>


---

Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/6724#-:-KHvHCI0c3vjxTTMNsrq:676834233)*

**Comment by tschottdorf** [comment]
Skipped the test (though I'm hoping the update I pushed along with that does the trick).
Will run this in loops on my personal fork and open a new PR when I'm pretty confident it's not flaky any more.

**Pull request merged**